include/openthread/link.h

```diff
@@ -1126,6 +1126,77 @@ uint8_t otLinkGetWakeupChannel(otInstance *aInstance);
  */
 otError otLinkSetWakeupChannel(otInstance *aInstance, uint8_t aChannel);
 
+/**
+ * Enables or disables listening for wake-up frames.
+ *
+ * Requires `OPENTHREAD_CONFIG_WAKEUP_END_DEVICE_ENABLE`.
+ *
+ * @param[in]  aInstance     A pointer to an OpenThread instance.
+ * @param[in]  aEnable       true to enable listening for wake-up frames, or false otherwise.
+ *
+ * @retval OT_ERROR_NONE          Successfully enabled / disabled the listening for wake-up frames.
+ * @retval OT_ERROR_INVALID_ARGS  Configured listen interval is not greater than listen duration.
```

Comment:
Could you revise this so it's not a negative?  Also, if they're both configurable (appears to be) I would suggest just dropping the word "configured".

`The listen interval is less than the listen duration.`

or

`The listen duration is greater than the listen interval.`